components/dialog/stories/dialog.stories.js

```diff
@@ -65,17 +120,150 @@ export default {
+ */
 export const Default = Template.bind({});
 Default.args = {
 	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
 	showModal: true,
 	content: [
-		() => Typography({
-			semantics: "body",
-			size: "m",
-			content: [
-				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl. Diam volutpat commodo sed egestas egestas. Dolor magna eget est lorem ipsum dolor. Vitae suscipit tellus mauris a diam maecenas sed. Turpis in eu mi bibendum neque egestas congue. Rhoncus est pellentesque elit ullamcorper dignissim cras lobortis."
-			]
-		}),
+		() => ExampleContent
+	],
+};
+
+/*
+ * Stories for "docs only."
+ */
+export const Small = Template.bind({});
+Small.tags = ["docs-only"];
+Small.parameters = {};
+Small.args = {
+	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	showModal: true,
+	size: "small",
+	content: [
+		() => ExampleContent
+	],
+};
+
+export const Large = Template.bind({});
+Large.tags = ["docs-only"];
+Large.parameters = {};
+Large.args = {
+	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	showModal: true,
+	size: "large",
+	content: [
+		() => ExampleContent
+	],
+};
+
+/**
+ * A dialog that can be dismissed without taking an actions. Dismissible dialogs should never have buttons.
+ */
+export const Dismissible = Template.bind({});
+Dismissible.tags = ["docs-only"];
+Dismissible.parameters = {};
+Dismissible.args = {
+	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	showModal: true,
+	isDismissible: true,
+	content: [
+		() => ExampleContent
+	],
+};
+
+/**
+ * Dialogs can forgo the divider if they have content that spans the entire width of the dialog.
+ */
+export const NoDivider = Template.bind({});
+NoDivider.tags = ["docs-only"];
+NoDivider.parameters = {};
+NoDivider.args = {
+	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	showModal: true,
+	isDismissible: true,
+	hasDivider: false,
+	content: [
+		() => ExampleContent
+	],
+};
+
+/**
+ * Dialogs can have a hero or cover image header.
+ */
+export const WithHero = Template.bind({});
+WithHero.tags = ["docs-only"];
+WithHero.parameters = {
+	docs: {
+		story: {
+			height: "650px",
+		},
+	},
+};
+WithHero.args = {
+	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	isDismissible: true,
+	showModal: true,
+	hasHeroImage: true,
+	heroImageUrl: "example-card-portrait.png",
+	content: [
+		() => ExampleContent
+	],
+};
+
+/**
+ * A dialog without `.spectrum-Dialog--alert` can expand to a larger size to house larger contents.
```

Comment:
I think all of the dialogs might now scroll if their contents are longer than their available space, and I didn't see `.spectrum-Dialog--alert` in the CSS anymore, I wonder if you could double-check if this comment needs updating?

Reply:
@rise-erpelding I wasn't sure how to handle this comment. I assume that was the class name before alert dialog was its own component. Is that a question for design? (that all dialogs scroll if their content is longer than the available space?) I know I'm mixing the two branches, but scrolling isn't represented in the S2 designs, so it might be a good question anyways.

Reply:
I think checking on the scrolling behavior whenever we get to it in S2 would be ok, I think it's probably out of scope for this work. And it seems like we might want it to scroll: 

> [The content inside the body area can scroll when necessary](https://spectrum-contributions.corp.adobe.com/page/standard-dialog-beta/#Body-area).

Quickly scanning the repo history it seems like this line in the documentation has been around for a [number of years](https://github.com/adobe/spectrum-css/blob/%40adobe/spectrum-css%402.14.0-alpha.10/components/dialog/docs.yml#L256) and it looks like that `.spectrum-Dialog--alert` class was used at one point but isn't anymore. I would say we could probably drop this comment/story description as it no longer seems needed, or if you still wanted to keep some text there, you could write "The content inside the body area can scroll when necessary."

Reply:
Ok, I added scrolling behavior to my list of questions for design for dialog S2. 👍

I used that sentence you suggested. https://github.com/adobe/spectrum-css/pull/2833/commits/e5abf1453cf4e803d67b88e4f0952338bbfd923b